students/Matt_Hudgins/Lesson05/mailroom_3.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+# chmod +x mailroom.py needs to be performed before executable
+
+'''
+    File Name: mailroom_3.py
+    Author: Matt Hudgins
+    Date created: 5/20/18
+    Date last modified: 5/20/18
+    Python Version 3.6.4
+'''
+import sys
+from collections import deque
+
+# This is the donor list
+donors = {}
+donors['Rick Grimes'] = [5.00, 10.00, 2.00]
+donors['Shane Walsh'] = [4.00, 10.00, 9.00]
+donors['Carl Grimes'] = [72.00, 10.00, 88.00]
+donors['Morgan Jones'] = [68.00, 10.00, 98.00]
+
```

Comment:
This would generally be defined as a dict literal using this syntax:
```
donors = {
    'Rick Grimes': [5.00, 10.00, 2.00],
    'Shane Walsh': [4.00, 10.00, 9.00],
}
```